routes/orders.js

```
@@ -96,6 +189,14 @@ router.post('/', function (req, res, next) {
     console.log(`Blocked email: ${email}`)
     return res.status(400).send('Orders from this email domain are not allowed.');
```

Let's return a generic error message here. We don't want bad actors to know why they're getting blocked. Also please return JSON with a `message` property instead of plain text.

---

Good idea, updated both places this check exists with that in mind.